fix build issues part 2

@claude my /.github/workflows/build-app.yml workflow is failing with the following errors:

---
<br class="Apple-interchange-newline">
build-and-commit: src/schemas/generated-uiSchema.ts#L18431Type '"conditional"' is not assignable to type '"simple" \| "complex" \| "provider"'.
--
build-and-commit: src/schemas/generated-uiSchema.ts#L18414Type '"conditional"' is not assignable to type '"simple" \| "complex" \| "provider"'.
build-and-commit: src/schemas/generated-uiSchema.ts#L18397Type '"conditional"' is not assignable to type '"simple" \| "complex" \| "provider"'.
build-and-commit: src/schemas/generated-uiSchema.ts#L18380Type '"conditional"' is not assignable to type '"simple" \| "complex" \| "provider"'.
build-and-commit: src/schemas/generated-uiSchema.ts#L18363Type '"conditional"' is not assignable to type '"simple" \| "complex" \| "provider"'.
build-and-commit: src/schemas/generated-uiSchema.ts#L18349Type '{ readonly AZURE_STORAGE_ENDPOINT: { readonly name: "AZURE_STORAGE_ENDPOINT"; readonly type: "string"; readonly 'ui:title': "Azure Storage Endpoint"; readonly 'ui:description': "Sets the endpoint URL for Azure Storage."; ... 7 more ...; readonly 'ui:extensions': { ...; }; }; ... 368 more ...; readonly TASK_MODEL_EXT...' is not assignable to type 'Record<string, FieldConfiguration>'.
build-and-commit: src/schemas/generated-schemas.ts#L447Cannot redeclare block-scoped variable 'OpenWebUIConfig'.
build-and-commit: src/schemas/generated-schemas.ts#L9Cannot redeclare block-scoped variable 'OpenWebUIConfig'.
build-and-commit: src/schemas/generated-schemas.ts#L8Cannot redeclare block-scoped variable 'OpenWebUIConfig'.
build-and-commit: src/components/ui/form/feedback/toast-error.tsx#L3Cannot find module '@/components/ui/use-toast' or its corresponding type declarations.

[build-and-commit: src/schemas/generated-uiSchema.ts#L18431](https://github.com/leger-labs/openwebui-config.com/pull/22/files#annotation_38369720696)
Type '"conditional"' is not assignable to type '"simple" | "complex" | "provider"'.
[build-and-commit: src/schemas/generated-uiSchema.ts#L18414](https://github.com/leger-labs/openwebui-config.com/pull/22/files#annotation_38369720704)
Type '"conditional"' is not assignable to type '"simple" | "complex" | "provider"'.
[build-and-commit: src/schemas/generated-uiSchema.ts#L18397](https://github.com/leger-labs/openwebui-config.com/pull/22/files#annotation_38369720711)
Type '"conditional"' is not assignable to type '"simple" | "complex" | "provider"'.
[build-and-commit: src/schemas/generated-uiSchema.ts#L18380](https://github.com/leger-labs/openwebui-config.com/pull/22/files#annotation_38369720720)
Type '"conditional"' is not assignable to type '"simple" | "complex" | "provider"'.
[build-and-commit: src/schemas/generated-uiSchema.ts#L18363](https://github.com/leger-labs/openwebui-config.com/pull/22/files#annotation_38369720728)
Type '"conditional"' is not assignable to type '"simple" | "complex" | "provider"'.
[build-and-commit: src/schemas/generated-uiSchema.ts#L18349](https://github.com/leger-labs/openwebui-config.com/pull/22/files#annotation_38369720734)
Type '{ readonly AZURE_STORAGE_ENDPOINT: { readonly name: "AZURE_STORAGE_ENDPOINT"; readonly type: "string"; readonly 'ui:title': "Azure Storage Endpoint"; readonly 'ui:description': "Sets the endpoint URL for Azure Storage."; ... 7 more ...; readonly 'ui:extensions': { ...; }; }; ... 368 more ...; readonly TASK_MODEL_EXT...' is not assignable to type 'Record<string, FieldConfiguration>'.
[build-and-commit: src/schemas/generated-schemas.ts#L447](https://github.com/leger-labs/openwebui-config.com/pull/22/files#annotation_38369720742)
Cannot redeclare block-scoped variable 'OpenWebUIConfig'.
[build-and-commit: src/schemas/generated-schemas.ts#L9](https://github.com/leger-labs/openwebui-config.com/pull/22/files#annotation_38369720749)
Cannot redeclare block-scoped variable 'OpenWebUIConfig'.
[build-and-commit: src/schemas/generated-schemas.ts#L8](https://github.com/leger-labs/openwebui-config.com/pull/22/files#annotation_38369720754)
Cannot redeclare block-scoped variable 'OpenWebUIConfig'.
[build-and-commit: src/components/ui/form/feedback/toast-error.tsx#L3](https://github.com/leger-labs/openwebui-config.com/pull/22/files#annotation_38369720760)
Cannot find module '@/components/ui/use-toast' or its corresponding type declarations.
----


fix the codebase so that we no longer run into these errors. definition of done is that i can manually run the build-app action successfully.

investigate the /scripts section as well, which are controlled with another gh action form-generation-pipeline.yml as some errors may come from these scripts. 
fix the codebase so that we no longer run into these errors. definition of done is that i can manually run the build-app action successfully.

--- a/.gitignore
+++ b/.gitignore
@@ -1,0 +1,85 @@
+# Node.js
+node_modules/
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Build output
+dist/
+build/
+.vite/
+
+# Environment files
+.env
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+# Editor files
+.vscode/
+.idea/
+*.swp
+*.swo
+
+# OS generated files
+.DS_Store
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+ehthumbs.db
+Thumbs.db
+
+# Logs
+logs
+*.log
+
+# Coverage directory used by tools like istanbul
+coverage/
+
+# nyc test coverage
+.nyc_output
+
+# Dependency directories
+jspm_packages/
+
+# Optional npm cache directory
+.npm
+
+# Optional eslint cache
+.eslintcache
+
+# Optional REPL history
+.node_repl_history
+
+# Output of 'npm pack'
+*.tgz
+
+# Yarn Integrity file
+.yarn-integrity
+
+# dotenv environment variables file
+.env
+
+# parcel-bundler cache (https://parceljs.org/)
+.cache
+.parcel-cache
+
+# next.js build output
+.next
+
+# nuxt.js build output
+.nuxt
+
+# vuepress build output
+.vuepress/dist
+
+# Serverless directories
+.serverless
+
+# FuseBox cache
+.fusebox/
+
+# DynamoDB Local files
+.dynamodb/

--- a/scripts/extract-uiSchema/dependency-processor.js
+++ b/scripts/extract-uiSchema/dependency-processor.js
@@ -56,7 +56,7 @@ function extractDependencyRelationships(extensionsMap) {
     if (dependsOn) {
       relationships.push({
         field: fieldName,
-        type: 'conditional',
+        type: 'simple',
         source: 'depends-on',
         dependency: dependsOn,
         metadata: fieldData.standardMetadata

--- a/src/components/ui/form/feedback/toast-error.tsx
+++ b/src/components/ui/form/feedback/toast-error.tsx
@@ -1,6 +1,6 @@
 "use client"
 
-import { useToast } from "@/components/ui/use-toast"
+import { useToast } from "@/hooks/use-toast"
 import { ToastAction } from "@/components/ui/toast"
 import { AlertCircle } from "lucide-react"
 

--- a/src/schemas/generated-schemas.ts
+++ b/src/schemas/generated-schemas.ts
@@ -5,7 +5,6 @@
 import { z } from 'zod';
 
 // Individual schemas for each configuration property
-export type OpenWebUIConfig = z.infer<typeof OpenWebUIConfig>;
 export const OpenWebUIConfig = z.object({
   WEBUI_URL: z.string().optional(),
   ENABLE_SIGNUP: z.boolean().optional(),

--- a/src/schemas/generated-uiSchema.ts
+++ b/src/schemas/generated-uiSchema.ts
@@ -5,7 +5,7 @@
  * This file contains UI organization metadata extracted from the OpenAPI schema
  * for use with RJSF-compatible form generation and react-ts-form integration.
  *
- * Generated at: 2025-08-28T18:03:09.323Z
+ * Generated at: 2025-08-29T05:18:27.959Z
  * Source: OpenWebUI OpenAPI Configuration
  *
  * Statistics:
@@ -18360,7 +18360,7 @@ export const fieldConfigurations: Record<string, FieldConfiguration> = {
 // Conditional Rules - Defines when fields should be shown/hidden
 export const conditionalRules: Record<string, ConditionalRule> = {
   WEBHOOK_URL: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18377,7 +18377,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   ADMIN_EMAIL: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18394,7 +18394,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OLLAMA_BASE_URLS: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18411,7 +18411,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OPENAI_API_BASE_URL: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18428,7 +18428,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OPENAI_API_BASE_URLS: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18445,7 +18445,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OPENAI_API_KEY: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18462,7 +18462,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OPENAI_API_KEYS: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18479,7 +18479,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   TITLE_GENERATION_PROMPT_TEMPLATE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18496,7 +18496,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   CODE_EXECUTION_ENGINE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18628,7 +18628,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   CODE_INTERPRETER_ENGINE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18645,7 +18645,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   CODE_INTERPRETER_PROMPT_TEMPLATE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18777,7 +18777,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   AUTOCOMPLETE_GENERATION_INPUT_MAX_LENGTH: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18794,7 +18794,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   AUTOCOMPLETE_GENERATION_PROMPT_TEMPLATE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18811,7 +18811,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   TAGS_GENERATION_PROMPT_TEMPLATE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18828,7 +18828,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   API_KEY_ALLOWED_ENDPOINTS: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18845,7 +18845,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   WEBUI_SESSION_COOKIE_SAME_SITE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18862,7 +18862,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   WEBUI_SESSION_COOKIE_SECURE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18879,7 +18879,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   WEBUI_AUTH_COOKIE_SAME_SITE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18896,7 +18896,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   WEBUI_AUTH_COOKIE_SECURE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18913,7 +18913,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   WEBUI_SECRET_KEY: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18930,7 +18930,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   RAG_EMBEDDING_MODEL_TRUST_REMOTE_CODE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -18947,7 +18947,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   RAG_RERANKING_MODEL_TRUST_REMOTE_CODE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -20132,7 +20132,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   RAG_TOP_K: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -20149,7 +20149,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   RAG_TOP_K_RERANKER: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -20166,7 +20166,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   RAG_RELEVANCE_THRESHOLD: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -20287,7 +20287,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   RAG_RERANKING_MODEL: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -20419,7 +20419,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   QUERY_GENERATION_PROMPT_TEMPLATE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -20436,7 +20436,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   GOOGLE_DRIVE_CLIENT_ID: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -20453,7 +20453,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   GOOGLE_DRIVE_API_KEY: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -20470,7 +20470,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   ONEDRIVE_CLIENT_ID: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -20487,7 +20487,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   ENABLE_SEARCH_QUERY_GENERATION: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -20504,7 +20504,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   WEB_SEARCH_TRUST_ENV: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -20521,7 +20521,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   WEB_SEARCH_RESULT_COUNT: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -20538,7 +20538,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   WEB_SEARCH_CONCURRENT_REQUESTS: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -20555,7 +20555,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   WEB_SEARCH_ENGINE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -20572,7 +20572,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   BYPASS_WEB_SEARCH_EMBEDDING_AND_RETRIEVAL: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -21659,7 +21659,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   IMAGE_GENERATION_ENGINE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -21676,7 +21676,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   ENABLE_IMAGE_PROMPT_GENERATION: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -21693,7 +21693,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   IMAGE_PROMPT_GENERATION_PROMPT_TEMPLATE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -21710,7 +21710,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   IMAGE_SIZE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -21756,7 +21756,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   IMAGE_GENERATION_MODEL: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22095,7 +22095,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OAUTH_MERGE_ACCOUNTS_BY_EMAIL: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22112,7 +22112,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OAUTH_UPDATE_PICTURE_ON_LOGIN: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22129,7 +22129,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   WEBUI_AUTH_TRUSTED_EMAIL_HEADER: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22146,7 +22146,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   WEBUI_AUTH_TRUSTED_NAME_HEADER: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22163,7 +22163,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   GOOGLE_CLIENT_ID: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22180,7 +22180,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   GOOGLE_CLIENT_SECRET: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22197,7 +22197,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   GOOGLE_OAUTH_SCOPE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22214,7 +22214,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   GOOGLE_REDIRECT_URI: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22231,7 +22231,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   MICROSOFT_CLIENT_ID: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22248,7 +22248,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   MICROSOFT_CLIENT_SECRET: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22265,7 +22265,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   MICROSOFT_CLIENT_TENANT_ID: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22282,7 +22282,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   MICROSOFT_OAUTH_SCOPE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22299,7 +22299,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   MICROSOFT_REDIRECT_URI: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22316,7 +22316,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   GITHUB_CLIENT_ID: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22333,7 +22333,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   GITHUB_CLIENT_SECRET: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22350,7 +22350,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   GITHUB_CLIENT_SCOPE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22367,7 +22367,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   GITHUB_CLIENT_REDIRECT_URI: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22384,7 +22384,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OAUTH_CLIENT_ID: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22401,7 +22401,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OAUTH_CLIENT_SECRET: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22418,7 +22418,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OPENID_PROVIDER_URL: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22435,7 +22435,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OPENID_REDIRECT_URI: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22452,7 +22452,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OAUTH_SCOPES: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22469,7 +22469,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OAUTH_CODE_CHALLENGE_METHOD: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22486,7 +22486,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OAUTH_PROVIDER_NAME: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22503,7 +22503,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OAUTH_USERNAME_CLAIM: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22520,7 +22520,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OAUTH_EMAIL_CLAIM: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22537,7 +22537,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OAUTH_PICTURE_CLAIM: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22554,7 +22554,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OAUTH_GROUP_CLAIM: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22571,7 +22571,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OAUTH_ROLES_CLAIM: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22588,7 +22588,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OAUTH_ALLOWED_ROLES: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22605,7 +22605,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OAUTH_ADMIN_ROLES: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22622,7 +22622,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   OAUTH_ALLOWED_DOMAINS: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22639,7 +22639,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   LDAP_SERVER_LABEL: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22656,7 +22656,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   LDAP_SERVER_HOST: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22673,7 +22673,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   LDAP_SERVER_PORT: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22690,7 +22690,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   LDAP_ATTRIBUTE_FOR_MAIL: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22707,7 +22707,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   LDAP_ATTRIBUTE_FOR_USERNAME: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22724,7 +22724,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   LDAP_APP_DN: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22741,7 +22741,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   LDAP_APP_PASSWORD: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22758,7 +22758,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   LDAP_SEARCH_BASE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22775,7 +22775,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   LDAP_SEARCH_FILTER: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22792,7 +22792,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   LDAP_SEARCH_FILTERS: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22809,7 +22809,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   LDAP_USE_TLS: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22826,7 +22826,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   LDAP_CA_CERT_FILE: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22843,7 +22843,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   LDAP_VALIDATE_CERT: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22860,7 +22860,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   LDAP_CIPHERS: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22877,7 +22877,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   USER_PERMISSIONS_CHAT_TEMPORARY_ENFORCED: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22894,7 +22894,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   USER_PERMISSIONS_WORKSPACE_MODELS_ALLOW_PUBLIC_SHARING: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22911,7 +22911,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   USER_PERMISSIONS_WORKSPACE_KNOWLEDGE_ALLOW_PUBLIC_SHARING: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22928,7 +22928,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   USER_PERMISSIONS_WORKSPACE_PROMPTS_ALLOW_PUBLIC_SHARING: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -22945,7 +22945,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   USER_PERMISSIONS_WORKSPACE_TOOLS_ALLOW_PUBLIC_SHARING: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -23284,7 +23284,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   WEBSOCKET_MANAGER: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -23301,7 +23301,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   WEBSOCKET_REDIS_URL: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -23318,7 +23318,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   WEBSOCKET_SENTINEL_HOSTS: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -23335,7 +23335,7 @@ export const conditionalRules: Record<string, ConditionalRule> = {
     },
   },
   WEBSOCKET_SENTINEL_PORT: {
-    type: 'conditional',
+    type: 'simple',
     operator: 'and',
     rules: [
       {
@@ -26925,7 +26925,7 @@ export const reactTsFormIntegration = integrationMetadata.reactTsForm;
 export const openWebUiUiSchema: OpenWebUIUiSchema = {
   $schema: 'https://rjsf-team.github.io/react-jsonschema-form/docs/',
   $generated: {
-    generatedAt: '2025-08-28T18:03:09.323Z',
+    generatedAt: '2025-08-29T05:18:27.959Z',
     sourceSchema: 'OpenWebUI OpenAPI Configuration',
     totalFields: 370,
     totalCategories: 55,

--- a/src/schemas/index.ts
+++ b/src/schemas/index.ts
@@ -1,4 +1,33 @@
 // Export all schema-related modules
 export * from './generated-schemas'
 export * from './generated-uiSchema'
-export * from './generated-component-mapping'
+
+// Explicit exports from component mapping to avoid conflicts
+export { 
+  type ComponentMapping,
+  type ComponentProps,
+  type AvailableComponent,
+  type AvailableWrapper,
+  type ComponentRegistry,
+  type IntegrationMetadata,
+  type OpenWebUIComponentMappings,
+  componentMappings as formComponentMappings, // Rename to avoid conflict
+  componentProps as formComponentProps, // Rename to avoid conflict
+  fieldWrappers,
+  integrationMetadata as formIntegrationMetadata, // Rename to avoid conflict
+  getComponentForField,
+  getComponentProps as getFormComponentProps, // Rename to avoid conflict  
+  getFieldWrappers,
+  getAlternativeComponents,
+  isConditionalField as isFormConditionalField, // Rename to avoid conflict
+  isPremiumField,
+  getFieldsUsingComponent,
+  getComponentUsageStats,
+  getFieldsByCategory,
+  hasComponentMapping,
+  getComponentDecisionPath,
+  getComponentDecisionReason,
+  createReactTsFormConfig,
+  validateComponentMappings,
+  openWebUIComponentMappings
+} from './generated-component-mapping'

--- a/src/utils/env-converter.ts
+++ b/src/utils/env-converter.ts
@@ -121,11 +121,11 @@ export function convertEnvToForm(
     Object.entries(data).forEach(([key, value]) => {
       // Type conversion for known boolean fields
       if (isBooleanField(key)) {
-        processedData[key] = normalizeBooleanValue(value)
+        processedData[key] = normalizeBooleanValue(value || '')
       }
       // Type conversion for known numeric fields
       else if (isNumericField(key)) {
-        const numValue = normalizeNumericValue(value)
+        const numValue = normalizeNumericValue(value || '')
         if (numValue !== null) {
           processedData[key] = numValue.toString()
         } else {
@@ -135,7 +135,7 @@ export function convertEnvToForm(
       }
       // Handle URL fields
       else if (isUrlField(key)) {
-        const normalizedUrl = normalizeUrlValue(value)
+        const normalizedUrl = normalizeUrlValue(value || '')
         if (normalizedUrl !== value) {
           warnings.push(`${key} URL was normalized`)
         }

--- a/src/utils/validation.ts
+++ b/src/utils/validation.ts
@@ -1,6 +1,6 @@
 import { z } from 'zod'
 import type { ConfigData, ValidationError, EnvVariable } from '@/types'
-import { OpenWebUIConfig } from '@/schemas-original-output-from-gh-action/generated-schemas'
+import { OpenWebUIConfig } from '@/schemas/generated-schemas'
 
 // Validation result type
 export interface ValidationResult {
@@ -119,7 +119,7 @@ export function validateConfig(data: ConfigData): ValidationResult {
     const result = OpenWebUIConfig.safeParse(data)
     
     if (!result.success) {
-      result.error.errors.forEach(error => {
+      result.error.errors.forEach((error: any) => {
         errors.push({
           field: error.path.join('.') || 'root',
           message: error.message

